qabelbox/src/main/java/de/qabel/qabelbox/activities/MainActivity.java

```diff
@@ -228,6 +230,16 @@ private boolean uploadUri(Uri uri, String targetFolder) {
     protected void onCreate(Bundle savedInstanceState) {
 
         super.onCreate(savedInstanceState);
+        LocalQabelService service = QabelBoxApplication.getInstance().getService();
```

Comment:
I probably missed this in the last PR, so I ask now: Is it save to leak the service to other activities from the Application object? I thought every activity and service should bind directly to the service.